docs/UserGuide.md

```diff
@@ -565,9 +542,9 @@ Example: `nat/Singaporean`
 Example: `tg/T09`
 
 ### `s/SOCIAL_HANDLE`
-`SOCIAL_HANDLE` should be of the format `PLATFORM:USERID` and should adhere to the following constraints:
-* Only the following `PLATFORM` are supported: `Instagram`, `Telegram`, `Facebook`, `Twitter`, `Github`, `Linkedin`, `Snapchat`, `Discord`
-* The following 2 letter shorthand can be used to replace the full name of the social platform:
+`SOCIAL_HANDLE` should be of the format `PLATFORM:USERID` and adhere to the following constraints:
+* Only the following platforms are supported: `Instagram`, `Telegram`, `Facebook`, `Twitter`, `Github`, `Linkedin`, `Snapchat`, `Discord`
+* The following 2 letter shorthand represents the `PLATFORM` for their corresponding social platform:
```

Comment:
What `represents` implies may not be clear.

Reply:
Fixed. I changed the original sentence cos it implied that using the full word `Instagram` is supported by Socius.

Reply:
The full word is actually allowed.

Reply:
Reverting back to original line

Reply:
![image](https://user-images.githubusercontent.com/43673767/140697464-a32ed05f-3172-47af-bd16-e9fbd4cb5ae1.png)
Just to clarify, SOCIAL_HANDLE_USERID is accepted not SOCIAL_HANDLE?